src/anomalib/models/image/padim/lightning_model.py

```diff
@@ -51,18 +50,28 @@ def __init__(
         super().__init__()
 
         self.layers = layers
-        self.input_size = input_size
-        self.model: PadimModel = PadimModel(
-            input_size=input_size,
-            backbone=backbone,
-            pre_trained=pre_trained,
-            layers=layers,
-            n_features=n_features,
-        ).eval()
+        self.backbone = backbone
+        self.pre_trained = pre_trained
+        self.n_features = n_features
+        self.layers = layers
```

Comment:
I would be in favour of having `self.model: PadimModel` here even though it is not assigned. This will help keep a track of all the available attributes